### ISSUE TITLE
Deleting Private App Tags does not work as expected

### ISSUE THREAD

**Issue by orfl** [netskopeoss/terraform-provider-netskope#21]
I have a problem with the App Tag feature in Netskope's Terraform Provider. When I create app tags via terraform and later delete them via terraform, they still remain. Terraform reports that the App Tags have been deleted. But in the Netskope GUI they are still present. When I replace App Tags with other App Tags, they are replaced correctly. The problem exists only when deleting tags (so that after deleting no app tag is assigned to the private app).

Iam using Netskope Provider Version 0.2.4 and Terraform Version v1.5.2

Terraform CLI Output:
````
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # netskope_privateapps.PRIVAPP_Test will be updated in-place
  ~ resource "netskope_privateapps" "PRIVAPP_Test" {
        id                = "217"
        # (3 unchanged attributes hidden)

      + tags {
          + tag_name = "tag1"
        }
      + tags {
          + tag_name = "tag2"
        }
      + tags {
          + tag_name = "tag3"
        }
      + tags {
          + tag_name = "tag4"
        }

        # (4 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

netskope_privateapps.PRIVAPP_Test: Modifying... [id=217]
netskope_privateapps.PRIVAPP_Test: Modifications complete after 2s [id=217]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

.
.
.
.
.


Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # netskope_privateapps.PRIVAPP_Test will be updated in-place
  ~ resource "netskope_privateapps" "PRIVAPP_Test" {
        id                = "217"
        # (3 unchanged attributes hidden)

      - tags {
          - tag_name = "tag1" -> null
        }
      - tags {
          - tag_name = "tag2" -> null
        }
      - tags {
          - tag_name = "tag3" -> null
        }
      - tags {
          - tag_name = "tag4" -> null
        }

        # (4 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

netskope_privateapps.PRIVAPP_Test: Modifying... [id=217]
netskope_privateapps.PRIVAPP_Test: Modifications complete after 2s [id=217]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
````

**Comment by ns-sbrown** [comment]
Duplicate of [https://github.com/netskopeoss/terraform-provider-netskope/issues/7](https://github.com/netskopeoss/terraform-provider-netskope/issues/7)

**Comment by ns-sbrown** [comment]
Published v0.2.6 today’s  which resolves this issue.
